lib/screens/main_menu/menu_button.dart

```diff
@@ -0,0 +1,24 @@
+import 'package:flutter/material.dart';
+
+class MenuButton extends StatelessWidget {
+  final Function onPressed;
+  final String buttonText;
+
+  MenuButton(this.buttonText, {Function onPressed}):
+    this.onPressed = onPressed; 
```

Comment:
Think you can one-line this like `MenuButton(this.buttonText, {this.onPressed} )` 😄